Pipfile

```diff
@@ -10,16 +10,16 @@ tox = "*"
 flake8 = "*"
 isort = "*"
 black = "*"
-swimport = "*"
 
 [packages]
 numpy = "*"
 scipy = "*"
 scikit-learn = "*"
+cached-property = "*"
 pyside2 = "*"
 matplotlib = "*"
-cached-property = "*"
 pandas = "*"
+swimport = "*"
```

Comment:
I don't like swimport being necessary for running because it itself has other dependencies (lexicographic parsers for c++)

Reply:
from pipdeptree:
swimport==3.0.11
  - CppHeaderParser [required: >=2, installed: 2.7.4]
    - ply [required: Any, installed: 3.11]

I don't think that's too bad...

Reply:
I still don't understand why we don't also add the .cxx file and then both swimport and swig would be unnecessary for building...